app/assets/stylesheets/patternfly_overrides.scss

```diff
@@ -663,20 +663,9 @@ table.table.table-selectable tbody tr.selected td {
   color: #fff;
 }
 
-/* static class form-horizontal for displaying static info with reduced row height */
-.form-horizontal.static .form-group {
-  margin-bottom: 5px;
-  min-height: 28px;
-}
-
-/* resize png files displayed in form-groups */
-
-.form-horizontal.static .form-group img {
-  height: 20px;
-  width: 20px;
-}
+/* resize images displayed in form-group labels */
 
-.form-horizontal.static .form-group .control-labelimage img {
+.control-labelimage img {
```

Comment:
This one is only used in `app/views/catalog/_svccat_tree_show.html.haml`. Should we just move that width/height there?

(`.control-labelimage` also in `app/views/ops/_selected_by_servers.html.haml`, but without an image there)

Reply:
@himdel It's used in two places - I will do that. Thanks